### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by matthewdking**
#1

### CODE REVIEW

**Review by veredrec — CHANGES_REQUESTED** [review]
Please change the architecture image. Thanks! :wink:

**Review by veredrec — APPROVED** [review]

**Review by Heathercoraje — APPROVED** [review]

**Review by ghost — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,38 +1,92 @@
...
+<img alt="Prototype start and info page" src="https://user-images.githubusercontent.com/25408167/31076462-ddb5dd62-a782-11e7-993b-b03bd4b87c79.png" width=49%>
+<img alt="Prototype start and info page" src="https://user-images.githubusercontent.com/25408167/31076752-61ce4a70-a784-11e7-8542-998d451edd45.png" width=50%>
+<img alt="Prototype start and info page" src="https://user-images.githubusercontent.com/25408167/31076464-dde5ca54-a782-11e7-9d93-30a080092e30.png" width=25%>
+<img alt="Prototype start and info page" src="https://user-images.githubusercontent.com/25408167/31076466-ddead8dc-a782-11e7-8e45-ea310e21ac7a.png" width=25%>
 
 ### Architecture:
 
+![App architecture](https://user-images.githubusercontent.com/25408167/31123917-e141ee08-a84a-11e7-8524-57ca1ba9e4a5.jpg)
+
 ### Technologies:
```
Comment (Karyum) [inline README.md:59]: sort techs on what does what ?

File: README.md [hunk 2]
```diff
@@ -1,38 +1,92 @@
...
 
 ### User Story:
 ##### As a user I can...
-* read welcome page and instructions on how to use the app
-* view card with job and decide if I like / dislike it
-* click on card to view more details
-* View list of jobs I like
-* click on job to view details on the job and career path
-* read details instructions for the app in the help page
+* read welcome page and instructions on how to use the app.
```
Comment (Karyum) [inline README.md]: capitalize first letters for consistency

File: README.md [hunk 3]
```diff
@@ -1,38 +1,92 @@
...
-* all team member understand code before approval
+* CamelCase for JavaScript id's.
+* kebab-case for CSS Classes.
+* snake_case for multiple word file names.
+* comments for code clarification.
+* ES6 on backend and ES5 on the frontend.
+* all team members should understand code before approval (comments on PR).
+
+
+**Git Flow**
```
Comment (Karyum) [inline README.md]: :heart:

File: README.md [hunk 4]
```diff
@@ -1,38 +1,92 @@
...
 
 ### Team Guidelines:
 
-* CamelCase for JavaScript id's
-* kebab-case for CSS Classes
-* comments for code clarification
-* ES6 on backend
-* all team member understand code before approval
+* CamelCase for JavaScript id's.
+* kebab-case for CSS Classes.
```
Comment (Karyum) [inline README.md:78]: first letter capitalize

**Review by Karyum — CHANGES_REQUESTED** [review]
just some changes

File: README.md (hunk 2 above)
Comment (matthewdking) [reply README.md]: Yeah I agree :+1:

File: README.md (hunk 1 above)
Comment (matthewdking) [reply README.md:59]: sure thing

File: README.md (hunk 4 above)
Comment (matthewdking) [reply README.md:78]: kebab-case doesn't use a capital letter at the start because the name is an example of the way to write in kebab-case

Comment (Karyum) [reply README.md:78]: forgot to mention that CamelCase was the only one which was capitalized and thats pretty funny :laughing:

**Review by Karyum — APPROVED** [review]

**Pull request merged**